    @Test
    public void testCollectionToCollection() throws Exception {
        List<Integer> source = new ArrayList<>();
        source.add(1);
        source.add(2);

        Object converted =
                CollectionConverterFactory.CollectionToCollection.convert(source, Set.class);

        Assert.assertTrue(converted instanceof Set);
        Set target = (Set) converted;
        Assert.assertTrue(target.contains(1));
        Assert.assertTrue(target.contains(2));
    }

